docs/src/explanations/Connection-operators-.md

```diff
@@ -0,0 +1,399 @@
+---
+
+layout: docs
+
+title:  "Deep Dive into Connection Operators"
+
+section: "chisel3"
+
+---
+
+# Further Explanation for the := and  <> Operators
+
+Chisel contains two connection operators, `:=` and `<>`. This document provides a deeper explanation of the differences of the two and when to use one or the other. The differences are demonstrated with experiments using Scastie examples + DecoupledIO examples.
+
+
+
+### Experiment Setup
+
+```scala mdoc
+// Imports used by the following examples
+import chisel3._
+import chisel3.util.DecoupledIO
+```
+
+The diagram for the experiment can be viewed **[here](https://docs.google.com/document/d/14C918Hdahk2xOGSJJBT-ZVqAx99_hg3JQIq-vaaifQU/edit?usp=sharing)**
```

Comment:
```suggestion
The diagram for the experiment can be viewed [here](https://docs.google.com/document/d/14C918Hdahk2xOGSJJBT-ZVqAx99_hg3JQIq-vaaifQU/edit?usp=sharing)
```

I don't think we need this to be bold

Reply:
```suggestion
The diagram for the experiment can be viewed **[here](https://docs.google.com/document/d/14C918Hdahk2xOGSJJBT-ZVqAx99_hg3JQIq-vaaifQU/edit?usp=sharing)**

```

I don't think this was changed, although it's marked as resolved. I'm also adding a whitespace between this and the actual image.